storage2/src/storage.rs

```diff
@@ -59,15 +57,16 @@ impl Storage {
     /// `Storage`, or `None` if the tree is empty.
     pub async fn latest_version(&self) -> Result<Option<jmt::Version>> {
         // TODO: do better
-        Ok(latest_version(self.0.db).unwrap())
+        latest_version(&self.0.db)
```

Comment:
I wasn't sure what `do better` meant here, maybe it's about doing the read asynchronously?  One idea would be to read it out of the `latest_snapshot`, but I wasn't 100% clear that the existing code correctly handles the case where the tree is empty.